src/app.js

```diff
@@ -1 +1,57 @@
 'use strict';
+
+const fs = require('fs');
+const readline = require('readline');
+
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+function cp() {
+  const [sourse, destination] = process.argv.slice(2);
+
+  if (sourse === destination) {
+    rl.close();
+
+    return;
+  }
+
+  try {
+    fs.readFileSync(sourse);
+  } catch (error) {
```

Comment:
why exactly do you read here file? You do nothing after reading. If you try to call copyFileSync with the wrong source there will be an error too

Reply:
I tried to make check for file existence that way)